client/login/controller.js

```diff
@@ -24,7 +24,7 @@ import { getCurrentUser, getCurrentUserLocale } from 'state/current-user/selecto
 const enhanceContextWithLogin = context => {
 	const { path, params: { flow, isJetpack, socialService, twoFactorAuthType } } = context;
 
-	context.cacheQueryKeys = [ 'client_id' ];
+	context.cacheQueryKeys = [ 'client_id', 'signup_flow' ];
```

Comment:
FYI, This change is conflicting with https://github.com/Automattic/wp-calypso/pull/22614

Reply:
👍

Reply:
Thinking about it, it should not be a blocker.